AhmadDuhoki/css/style.css

```diff
@@ -0,0 +1,195 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html, body{
+    background-color: black;
+    color: #C0C0C0;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.logo{
+    width: 70px;
+    height: 65px; 
+    margin-top: 8px;
+    margin-left: 50px;
+}
+
+.header {
+    position: fixed;
+    top: 0;
+    z-index: 1;
+    width: 100%;
+    height: 80px;
+    background-color: black;
+    border-radius: 4px;
+    -webkit-box-shadow: 2px -10px 25px 3px rgba(255,166,0,1);
+    -moz-box-shadow: 2px -10px 25px 3px rgba(255,166,0,1);
+    box-shadow: 2px -10px 25px 3px rgba(255,166,0,1);
+}
+
+.light-btn{
+    border: 2px solid #FFA500;
+    padding: 6px 12px;
+    background-color: black;
+    color: #FFA500;
+    border-radius: 5px;
+}
+
+.light-btn:hover{
+    color: white;
+}
+
+.dark-btn{
+    border: 2px solid #FFA500;
+    padding: 6px 12px;
+    background-color: #FFA500;
+    color: black;
+    border-radius: 5px;
+}
+
+.dark-btn:hover{
+    color: white;
+}
+
+.btns {
+    float: right;
+    margin-top: 30px;
+    margin-right: 50px;
+}
+
+a{
+    text-decoration: none;
+    text-transform: uppercase;
+    color: white;
+}
+
+#wrapper{
```

Comment:
Try to stick with classes rather than ids when it comes to styling your webpage unless you really need to use an id.